pkg/devfile/parse_test.go

```diff
@@ -527,6 +527,90 @@ spec:
 				StarterProjects: map[string][]string{},
 			},
 		},
+		{
```

Comment:
perhaps add another test to test with context `./testdata/`, this was be supported by previous versions of library, need to ensure it's backward-compatible. 

the priority should be `devfile.yaml` > `.devfile.yaml` > `devfile.yml` > `.devfile.yml`